### ISSUE TITLE
[nautical] seamark:type=obstruction

### ISSUE THREAD

**Issue by borazslo** [osmandapp/Osmand#8644]
tl;dr; Please make icons/symbols for underwater obstructions in the nautical rendered view.

Our lake has a lot of underwater logs that can be dangerous for our sailboats. We have began to make surveys for exact coordinates for all these obstructions (points and areas as well). We alse collecting data about how deep these obstacles are under water (We would like to be able to have rendered these obstacles on the osmand android app so as to avoid accidents.

We (plan to) use: 
- seamark:type=obstruction
- seamark:obstruction:category=stump
- example: https://www.openstreetmap.org/node/6493220954

and/or:
- seamark:type=obstruction
- seamark:obstruction:category=foul_area 
- example: https://www.openstreetmap.org/way/691831883

+1 It would nice to print/render the water_level info as well, even though we have the data in meter: "0.8 m under water surface (when the official water level is 600 cm"
- seamark:obstruction:[water_level](https://wiki.openstreetmap.org/wiki/Seamarks/General_Attributes#Water_level_effect_.28WATLEV.29
)

( About seamark:obstruction:category:
https://wiki.openstreetmap.org/wiki/Key:seamark:obstruction:category )

**Comment by borazslo** [comment]
Sorry for the noob question but how can I make this change appear on my phone? 

Settings in OsmAnd:
  Download Maps -> Local -> World seamakrs = updated right now. 
  Download Maps -> Local -> Hungary = updated right now. 
  Plugins -> Nautical map: switched on
  Map style = Nautical.
Result: I can see seamarks (lights, etc) but not the new ones.

Other settings:
    Poi overlay - Nautical -> yes but
Result:  There are orange circles on the map and when I click on those I can find mz new seamark obstruction things. But all my other traditional seamarks have these orange circle.

OsmAnd 3.6.3, released: 2020-02-28 -> Do I need to wait for a new release?

I would like to see these new symbols on my map. Can you help out showing the right direction?

**Comment by xmd5a2** [comment]
Can't find any problem. This object is rendered as expected
https://www.openstreetmap.org/node/6493220954
![Screenshot from 2020-05-20 20-23-04](https://user-images.githubusercontent.com/5829014/82477204-d32fe900-9ad7-11ea-8ba1-2c34a3b41d20.png)
